### PR TITLE
update jellyfin 10.8.9 and dotnet to 6.0.405

### PR DESCRIPTION
**Pull request by mxwi**
## Description

- update jellyfin version to 10.8.9
- update dotnet to 6.0.405

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [x] Package update

### CODE REVIEW

File: spk/jellyfin/Makefile
Comment (mreid-tt) [inline spk/jellyfin/Makefile:17]: do you want to also consider updating the dotNET version to 6.0.405 similar to how the last update did? check out the file modifications at https://github.com/SynoCommunity/spksrc/pull/5438/files

Comment (mxwi) [reply spk/jellyfin/Makefile]: done

File: native/dotnet-sdk-6.0/Makefile
Comment (mreid-tt) [inline native/dotnet-sdk-6.0/Makefile:2]: this should be updated to:

# Version 6.0.13, SDK 6.0.405

File: spk/jellyfin/Makefile
Comment (mreid-tt) [inline spk/jellyfin/Makefile:17]: this should be updated to:

`CHANGELOG = "1. Update jellyfin to 10.8.9<br>2. Update dotnet 6.0.405"`

Comment (mxwi) [reply spk/jellyfin/Makefile]: done

File: native/dotnet-sdk-6.0/Makefile
Comment (mxwi) [reply native/dotnet-sdk-6.0/Makefile]: done

**Review by mreid-tt — APPROVED** [review]
great work! please ensure the other checklist tests are carried out once the build is complete

**Comment by mreid-tt** [comment]
@mxwi, thank you for your contribution to the SynoCommunity and welcome to you as a first-time code contributor. As a recommendation for future pull requests please refer to the [FAQ](https://github.com/SynoCommunity/spksrc/blob/master/CONTRIBUTING.md#pull-requests) where it says:

> Please create a new branch before pulling your request and avoid using the master branch.

This helps to ensure that the main branch remains clean and stable, and it also allows for easy collaboration and code review by other contributors.

**Comment by mreid-tt** [comment]
@publicarray, once the user has completed testing this PR should be good to merge. Any other things I should check before doing so?

**Comment by mxwi** [comment]
> @mxwi, thank you for your contribution to the SynoCommunity and welcome to you as a first-time code contributor. As a recommendation for future pull requests please refer to the [FAQ](https://github.com/SynoCommunity/spksrc/blob/master/CONTRIBUTING.md#pull-requests) where it says:
> 
> > Please create a new branch before pulling your request and avoid using the master branch.
> 
> This helps to ensure that the main branch remains clean and stable, and it also allows for easy collaboration and code review by other contributors.

I am happy to help. Testet with DSM 7.1 with x64 - success!

I already saw that my PR was looking a bit different than the others. I will take a look at it, thanks.

**Comment by mreid-tt** [comment]
> I already saw that my PR was looking a bit different than the others. I will take a look at it, thanks.

Should be okay for this PR. I'll give @publicarray a day to lend a second pair of eyes to this. So barring any issues, I'll merge your PR tomorrow morning.

**Comment by mreid-tt** [comment]
@stevenliuit, as the maintainer of this package do let me know if you have any objections to merging this PR.

**Comment by publicarray** [comment]
Hi thanks for updating the package. 

Personally I would refrain from adding more supported architectures as the low-end CPUs don't provide a good jellyfin experience if the user attempts to do any kind of transcoding. I think even streaming without transcoding they might struggle if they have other apps running due to low memory on those systems. (Especially if you go beyond 1080p video) Jellyfin is one of the more demanding packages but if enough users think it's useful than go ahead. I'd just expect tickets come in that say that it's not working as expected when it's just that they expect too much from such a low end system. My2c

**Comment by mreid-tt** [comment]
> Personally I would refrain from adding more supported architectures as the low-end CPUs don't provide a good jellyfin experience if the user attempts to do any kind of transcoding. I think even streaming without transcoding they might struggle if they have other apps running due to low memory on those systems. (Especially if you go beyond 1080p video) Jellyfin is one of the more demanding packages but if enough users think it's useful than go ahead. I'd just expect tickets come in that say that it's not working as expected when it's just that they expect too much from such a low end system. My2c

Thanks for the feedback. What I was going for was compatible archs. The current published versions (10.8.0-5 and 10.8.5-8) include `armada370` (10.8.0-5) as well as a number of ARMv7 archs (`alpine4k`, `armada375`, `armada38x` and `comcerto2k`) with builds for DSM6 (10.8.5-8). My intent was to remove these incompatible combinations from the build by not using the `DOTNET_UNSUPPORTED_ARCHS` group so they don't cause user issues as in the tickets I cited.

As for the user experience, yes you do have a valid point. Some forums have suggested that if users avoid transcoding by using native clients the experience should be more usable. We can always update the Wiki to point out this risk and link to Jellyfin resources regarding recommended hardware. I didn't want to exclude based on performance however since there are use cases of running Jellyfin on a Raspberry Pi. If however the potential for tickets is big concern I could just remove all support for ARMv7 as that would simplify things.

**Comment by mreid-tt** [comment]
@hgy59, when this package is published I would recommend un-publishing [version 10.8.0-5](https://github.com/SynoCommunity/spksrc/pull/5311) (which includes a build for `armada370`) as well as [version 10.8.5-8](https://github.com/SynoCommunity/spksrc/pull/5438) (which includes DSM6 builds for some ARMv7 archs), as these builds are incompatible with dotNET 6 (https://github.com/SynoCommunity/spksrc/issues/5574).

**Comment by hgy59** [comment]
@mreid-tt just to be sure: does jellyfin not run on some ARMv7 archs with DSM 7.x?

**Comment by mreid-tt** [comment]
> @mreid-tt just to be sure: does jellyfin not run on some ARMv7 archs with DSM 7.x?

@hgy59, at the moment after discussion with @publicarray, all support for ARMv7 archs was removed from the build.

While there is the possibility of running on ARMv7 with DSM 7.x, the transcoding will be very slow and likely result in a poor user experience. To draw a comparison from Plex, no ARMv7-based Synology platforms support transcoding. So for this package specifically we have opted to remove ARMv7.

**Comment by mreid-tt** [comment]
> @mreid-tt just to be sure: does jellyfin not run on some ARMv7 archs with DSM 7.x?

@hgy59, I believe that based on it being a dotNET 6 core app, it should be able to run on ARMv7 archs with DSM 7.x with the exception of `armada370`. Because there were earlier releases which included incompatible combinations, I have requested these be un-published. For this release we have removed all support for ARMv7 archs based on the expected user experience.

**Comment by mreid-tt** [comment]
@hgy59, thanks for publishing this. please remember to remove [version 10.8.5-8](https://github.com/SynoCommunity/spksrc/pull/5438) (which includes DSM6 builds for some ARMv7 archs) from the repository.

**Comment by hgy59** [comment]
> @hgy59, thanks for publishing this. please remember to remove [version 10.8.5-8](https://github.com/SynoCommunity/spksrc/pull/5438) (which includes DSM6 builds for some ARMv7 archs) from the repository.

I hope not to forget this, but I have to wait for 48h after publishing to be sure the new package versions got visible in the DSM Package Manager.

**Comment by hgy59** [comment]
@mreid-tt now I deleted the latest unsupported Jellyfin release in the repo. Only  v10.8.9-9  is available now.

**Comment by pjgpinto** [comment]
Hi, @mreid-tt are you able to create a build for armada38x? thank you

**Comment by pjgpinto** [comment]
@mreid-tt thank you for the explanation. I could see the armada37 build, I thought they were similar.
there is any way I can get an older build for DSM6?

**Comment by mreid-tt** [comment]
@pjgpinto, the `armada37xx` you saw is a Marvell A3720 CPU which is 64-bit unit (see [here](https://kb.synology.com/en-me/DSM/tutorial/What_kind_of_CPU_does_my_NAS_have)). Based on the discussion above, older 32-bit processors do not support transcoding well and would provide a poor user experience. As such these processors were removed from SynoCommunity support.

**Comment by mreid-tt** [comment]
@hgy59, I was looking at the Jellyfin versions published and noted you removed every version but the current one. I don't know if it's possible to restore the Version 10.7.7-4 which was built with a dotnet version <6.0 which should not have any of the incompatibility issues in the builds I asked to be de-activated but still run on older Synology hardware?

**Pull request merged**
